dokuwiki-text/src/test/resources/dokuwikitext/dokuwiki-contents/conf/acl.auth.php

```diff
@@ -1,10 +0,0 @@
-# acl.auth.php
```

Comment:
Would this file be interesting to import rights ?

Reply:
Yes, it's the right file. https://www.dokuwiki.org/acl